### PR TITLE
[sui-studio] improve generate

### PR DESCRIPTION
**Pull request by davidbarna**
## Description
* [x] fix indentatio in generate command
* [x] improve README template for generate command

## Related Issue
#114

### CODE REVIEW

File: packages/sui-studio/bin/sui-studio-generate.js [hunk 1]
```diff
@@ -67,79 +67,90 @@ Promise.all([
...
-  `
-  ),
+node_modules`),
 
   writeFile(
   COMPONENT_PACKAGE_NPMIGNORE_FILE,
-  `src
-  `
-  ),
+  `src`),
```
Comment (julian-gernun) [inline packages/sui-studio/bin/sui-studio-generate.js]: maybe you could add the assets folder

Comment (carlosvillu) [reply packages/sui-studio/bin/sui-studio-generate.js]: which is the assets folder ?!

Comment (julian-gernun) [reply packages/sui-studio/bin/sui-studio-generate.js]: we are adding previews (screenshots) for the readme in the assets folder

Comment (davidbarna) [reply packages/sui-studio/bin/sui-studio-generate.js]: It's just a comment (optional)

**Pull request merged**